wafer_space/projects/tasks.py

```
@@ -894,21 +1049,109 @@ def _apply_post_download_processing(
+        project_file: The project file being processed
+        content: The content to process
+        temp_path: Temporary file path for intermediate processing
+
+    Returns:
+        tuple: (processed_content, md5_hash, sha1_hash)
+
+    Raises:
+        ValueError: If pipeline processing fails or format validation fails
+    """
+    logger = logging.getLogger(__name__)
+    from django.conf import settings  # noqa: PLC0415
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)